gen.py

```diff
@@ -263,13 +380,13 @@ def empty_holiday(**kwargs):
     with open("src/data/mod.rs", "w") as f:
         rendered = env.from_string(country_mod).render(countries=countries)
         f.write(rendered)
-        
+
     for country in countries:
-        with open("src/data/{}.rs".format(country.code.lower()), "w") as f:
-            holiday = getattr(holidays, country.code, None)
+        with open("src/data/{}.rs".format(mod_name(country)), "w") as f:
+            # Could use `getattr(holidays, country.code, {}).subdivisions` but this only has the codes and not the names.
```

Comment:
Can you please elaborate on this comment? :thinking:

Reply:
`getattr(holidays, country.code, {}).subdivisions` can be used to get all the subdivision codes, but no way to get the full names we need. This would have been nice because we wouldn't need to define the subdivs in the `countries` array. I will remove the comment